syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/symptom.proto";
import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetSymptomNameRequest {
    uint32 symptom_id = 1;
}

message GetSymptomNameResponse {
	string symptom_name = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}